robottelo/ui/contentviews.py

```diff
@@ -450,6 +465,37 @@ def add_remove_errata_to_filter(self, cv_name, filter_name,
         else:
             self.click(locators['contentviews.remove_errata'])
 
+    def edit_erratum_date_range_filter(
+            self, cv_name, filter_name, errata_types=None, date_type='updated',
+            start_date=None, end_date=None, open_filter=True):
+        """Edit Erratum Date Range Filter"""
+        allowed_errata_types = ['security', 'enhancement', 'bugfix']
+        allowed_date_types = ['updated', 'issued']
+        if not errata_types:
+            # one errata type is mandatory
+            errata_types = ['security']
```

Comment:
this function will be changed for None value to be not updated,
but still if errata_types is not None it has to have one element as one element in the list is mandatory (open the filter and you will realize the UI will not allow all the check boxes to be unchecked)